CONTRIBUTING.md

```diff
@@ -56,19 +72,54 @@ python -m tox -e windows-dev # (Windows)
 
 This will
 
-1. Create a virtual environment located at `.tox/dev`.
-2. Install all the dependencies for PyBaMM, including the ones for documentation and development.
-3. Tell Python to use your local pybamm files when you use `import pybamm` anywhere on your system.
+a. Create a virtual environment located at `.tox/dev`.
+
+b. Install all the dependencies for PyBaMM, including the ones for documentation and development.
 
-Finally, activate your environment.
+c. Tell Python to use your local pybamm files when you use `import pybamm` anywhere on your system.
+
+5. Finally, activate your environment.
 
 
 ```bash
 source .tox/dev/bin/activate # (GNU/Linux and MacOS)
 #
-.tox\dev\Scripts\activate.bat # (Windows)
+.tox\windows-dev\Scripts\activate.bat # (Windows)
+```
+6. Inside the virtual environment, type:
+```bash
+pip install tox # (Skip this step if you used pipx above)
+```
+
+### 2. Without using tox
+1. Install virtual environment using the command below:
+```bash
+pip install virtualenv
+```
+2. Create a virtual environment `env` within your current directory using the command below:
+```bash
+python -m venv env 
+# (You can then “activate” the environment using the command below)
+env\Scripts\activate.bat
+# (This whole step can be ignored if you don't want a virtual environment)
 ```
 
+3. Installing PyBaMM
+
+This can be acieved by forking the repository and cloning it in your machine.
+
+Or type:
+```bash
+pip install pybamm
```

Comment:
Sundials should be installed _before_ installing PyBaMM. This is because the library is required to compiled the `idaklu` module (in `solvers/c_solvers/idaklu.cpp`).
Without sundials installed, `pip install pybamm` works fine, but only installs part of PyBaMM.

On mac, sundials is installed easily: just `brew install sundials`. But on current GNU/linux distributions, it must be compiled from source. This should change in the future as more a recent version of sundials in packaged in the repos.
Compiling sundials form source may be a bit intimidating for new users, and that's the reason why `tox` is recommended `tox -e pybamm-requires` automatically compiles and installs sundials

Reply:
So should I create different steps for both users? I'll add that mac users can use tox -e pybamm-requires too

Reply:
if I remember correctly, using `tox -e pybamm-requires` on mac leads to some issues down the line. Some dependencies are linked incorrectly. I never managed to figure out what exactly was happening, but in the end, mac users should be fine with installing sundials from Homebrew.
So, in summary:
- GNU/Linux: `tox -e pybamm-requires`
- MacOS: `brew install sundials` (maybe add that this is recommended instead of using tox)
- Windows: PyBaMM does not support Sundials yet for Windows.

Reply:
I added this in the recent commit but I did not include the ```tox -e pybamm-requires``` for mac users as I thought it wasn't required. Should I add it now?

Reply:
No, you did good - `tox -e pybamm-requires` isn't required for mac users.